quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -781,10 +781,15 @@ function createPki {
   pki_create_intermediate "${ZITI_SPURIOUS_INTERMEDIATE}" "${ZITI_SIGNING_INTERMEDIATE_NAME}" 1
 
   echo " "
-  pki_allow_list="${ZITI_CONTROLLER_HOSTNAME},localhost,127.0.0.1"
-  if [[ "$EXTERNAL_DNS" != "" ]]; then pki_allow_list="$pki_allow_list,$EXTERNAL_DNS"; fi
-  pki_client_server "${pki_allow_list}" "${ZITI_CONTROLLER_INTERMEDIATE_NAME}" "${ZITI_CONTROLLER_IP_OVERRIDE-}" "${ZITI_CONTROLLER_HOSTNAME}"
-  pki_client_server "${ZITI_EDGE_CONTROLLER_HOSTNAME},localhost,127.0.0.1" "${ZITI_EDGE_CONTROLLER_INTERMEDIATE_NAME}" "${ZITI_EDGE_CONTROLLER_IP_OVERRIDE-}" "${ZITI_EDGE_CONTROLLER_HOSTNAME}"
+  pki_allow_list_dns="${ZITI_CONTROLLER_HOSTNAME},localhost,$(hostname)"
```

Comment:
ZITI_CONTROLLER_HOSTNAME is commented out in .env. Do you mean to if check this here?

Reply:
not for this one. i just renamed pki_allow_list to pki_allow_list_dns where it was already `pki_allow_list="${ZITI_CONTROLLER_HOSTNAME},localhost,127.0.0.1"` . I just kept this one the same as it was before. I did change from the IP 127.0.0.1 to `$(hostname)` though.